nessie/src/test/java/org/apache/iceberg/nessie/TestNamespace.java

```diff
@@ -106,7 +106,7 @@ public void testCreatingAndDroppingNamespaceWithContent() throws NessieNotFoundE
 
     Assertions.assertThatThrownBy(() -> catalog.dropNamespace(namespace))
         .isInstanceOf(NamespaceNotEmptyException.class)
-        .hasMessage("Namespace 'test' is not empty. One or more tables exist.");
+        .hasMessageContaining("Namespace 'test' is not empty");
```

Comment:
Message was misleading. Can be something else than tables.

Reply:
right